src/canopy_server/app.py

```diff
@@ -332,7 +332,15 @@ def _init_engines():
         Tokenizer.initialize()
         kb = KnowledgeBase(index_name=index_name)
         context_engine = ContextEngine(knowledge_base=kb)
-        llm = OpenAILLM()
+        if llm == "OpenAILLM":
+            llm = OpenAILLM()
+        elif llm == "AzureOpenAILLM":
+            llm = AzureOpenAILLM(
+                api_key=os.getenv("OPENAI_API_KEY"),
+                base_url=os.getenv("OPENAI_BASE_URL"),
+            )
+        else:
+            raise ValueError("Config issue: LLM must be either OpenAILLM or AzureOpenAILLM")
```

Comment:
Config loader should automatically handle values, no need to have it here, in fact this will block other LLM implementations